rotkehlchen/tests/conftest.py

```diff
@@ -170,15 +173,7 @@ def vcr_config(request: pytest.FixtureRequest, default_cassette_name: str) -> di
     ^^^ this allows our fork of pytest-deadfixtures to ignore this fixture for usage detection
     since it cannot detect dynamic usage (request.getfixturevalue) in pytest-recording
     """
-    rewrite_path = os.environ.get('CASSETTE_REWRITE_PATH')
-    cassette_dir = get_cassette_dir(request)
-    cassette_path = cassette_dir / default_cassette_name
-
-    if rewrite_path and (cassette_path.is_relative_to(rewrite_path) or rewrite_path == '*'):
-        record_mode = 'rewrite'
-    else:
-        record_mode = 'once'
-
+    record_mode = 'once'
```

Comment:
Hmmm does it make sense to have this as an opion anymore then? Remove it completely. Also the docstring needs adjusting. I don't know how vcr.py works to judge what is correct and what not. I just need it to work both locally and in the CI